Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -161,11 +161,18 @@ def retrieve_and_display_info(formula_or_cask, name, repositories, args:)
     livecheck_latest = livecheck_result(formula_or_cask)
     pull_requests = retrieve_pull_requests(formula_or_cask, name) unless args.no_pull_requests?
 
+    print_name = begin
+      "#{name} (cask)" if formula_or_cask.is_a?(Cask::Cask) && Formula[name] && !args.cask?
+    rescue
```

Comment:
I know this isn't the only instance of this, but I'd try make this more specific, like `rescue FormulaUnavailableError`.

Reply:
Changed this and the other instance to be specific.